src/luarocks/cmd.lua

```diff
@@ -589,6 +597,9 @@ function cmd.run_command(description, commands, external_namespace, ...)
    -- Now that the config is fully loaded, reinitialize fs using the full
    -- feature set.
    fs.init()
+   if hardcoded.FORCE_HARDCODED then
+      lua_found = true
```

Comment:
What if there lua interpreter is not set in the hardcoded file?

Reply:
Added check for hardcoded.LUA